Clima/Controller/WeatherViewController.swift

```diff
@@ -78,17 +78,14 @@ extension WeatherViewController: UITextFieldDelegate {
     }
     //テキストフィールドが空文字列まま検索が行われようとしている時に、何をするか？
     func textFieldShouldEndEditing(_ textField: UITextField) -> Bool {
-        
+      
         //早期リターンで前提条件を満たしているかチェックし、満たしていなけばreturnで抜ける
         //テキストフィールドが空文字列でなければ検索を続行
-        if textField.text != "" {
-            return true
-        } else {
+        guard textField.text!.isEmpty else { return true }
```

Comment:
textField.textが強制アンラップ!になってしまっているので、以下にようにすると強制アンラップを回避できます。

```suggestion
        guard let text = textField.text, text.isEmpty else { return true }
```